src/main/target/common.h

```diff
@@ -17,12 +17,27 @@
  
 #pragma once
 
-#define CONFIG_FASTLOOP_PREFERRED_ACC 1
-
 #ifdef STM32F4
```

Comment:
@garthy, there is some merit in having a `common_stm32f[3|4].h` file, but there are also some disadvantages - namely that it encourages people to blindly put f4/f4 related `#defines` in that file, whereas some of these `#defines` might me related to other things, e.g. FLASH_SIZE, the number of ports or other factors that are related to the processor type but not directly defined by it.

So for the moment I think it is best to just use `common.h`. We can revisit this later when we have a larger number of constants defined and so can classify them according to their attributes.

Of course there is no reason why the `#ifdef stm32f4` constants in `config.h` should not move to `common.h`